lib/dbfuncprisma.js

```diff
@@ -138,6 +138,36 @@ export async function updatePageData(data) {
   }
 }
 
+export async function restoreBackupData(data) {
+  if (!data) throw new Error("no data to backup");
+  // get old data as temporal backup
+  let oldData = await getPageDatawLinkAndSocialData();
+
+  // delete old data
+  await Promise.all([
+    Prisma.linkdata.deleteMany(),
```

Comment:
```js
 await Prisma.$executeRaw("ALTER SEQUENCE linkdata_id_seq RESTART WITH 1;");
```
i thinks it's better to reset the id sequence also because we are backing up with id
better to do this with social icons also.

Reply:
yes!, I had not taken it into account
fixed in 52e36cb6fd9d94028efd06419571a0cb386791bc